### ISSUE TITLE
QA Ph2_Accessibility: Home - Missing/skipped heading

### ISSUE THREAD

**Issue by Remiqa** [GoogleChrome/ioweb2016#382]
There are a number of alerts on the home page when using the WAVE extension for Chrome, set to filter WCAG 2.0 AA.
-1 case of missing first level heading
-3 cases of skipped heading level

Please note: I'm unable to get the code feature on WAVE to expand to give a detailed screenshot of where the issues are found.

Steps: 
1. Go to https://phase2-dot-io-webapp-staging.appspot.com/io2016/
2. Enable the WAVE extension
3. View Style alerts with the filter set to WCAG 2.0 AA 
4. Note 4 alerts 

Browser: 
Chrome 48 Mac OS X 10.11 

![gio_382a](https://cloud.githubusercontent.com/assets/7585604/13580256/3d750116-e456-11e5-89ba-8e14def45a0a.png)
![gio_382b](https://cloud.githubusercontent.com/assets/7585604/13580257/3d880af4-e456-11e5-976e-03590be259bf.png)
